### DISCUSSION TITLE
Pixelization and lines

### DISCUSSION THREAD

**Discussion by infog-raphY** [lkwq007/stablediffusion-infinity#59] [Q&A] [answered]
Hi guys and girls, 

I wanted to know if there are any tips to prevent our image from pixelating as it expands?
It also creates lines between the different extensions, it's a shame. Any way to avoid this?

I'm using the version on Google Colab.

[Here an exemple](https://i.imgur.com/7s7HRWG.png)
Look at the face of the girl on the center. More I work on it, more she pixelate.
And look next to the Snowman, it's pinky and lines everywhere.

Thanks for your help.

Another question: Could you explain me the differences between the "init mode"?

Otherwise, I have so much fun with SD-infinity, thanks for your fantastic work!

**Comment by lkwq007** [comment, marked as answer]
Related https://github.com/lkwq007/stablediffusion-infinity/issues/15
> I wanted to know if there are any tips to prevent our image from pixelating as it expands?

Turn on photometric correction. Note that it only works on Linux/MacOS at present. 

> Could you explain me the differences between the "init mode"?

Init model determine how missing regions are filled before passed into stable diffusion pipeline. Usually `patchmatch` can get better results. `edge_pad` can generate seamless results but might get more artifacts. other `init_mode`s might not work well
